caluma/caluma_workflow/tests/test_work_item.py

```diff
@@ -336,6 +336,7 @@ def test_complete_multiple_instance_task_form_work_item_next(
     snapshot.assert_match(result.data)
 
 
+@pytest.mark.flaky(10, 9)
```

Comment:
Does this mean "at least 9 successes in 10 runs"?

Reply:
Yeah, I just wanted to try out if flaky helps us in any way... But doesn't seem to be the case, we don't want any flakiness.